### PR TITLE
[frontend] Note & Opinion displayed as unknown in relathionship creation form (#9245)

### LINKED ISSUE #9245: Note & Opinion displayed as unknown in relationship creation form
## Description
Note displayed as unknown in relationship creation form

## Reproducible Steps
- Go to a malware
- Go to the knowledge tab
- Click on 'related entities' and 'create'
- in the list, select a note
Image

- In the form, the note is displayed as unknown
Image

## Comment
Same for opinions

### PR DESCRIPTION
**Pull request by ValentinBouzinFiligran**
### Proposed changes

* getMainRepresentative added

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/9245

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx [hunk 1]
```diff
@@ -708,11 +708,14 @@ const StixCoreRelationshipCreationFromEntity: FunctionComponent<StixCoreRelation
   } = useEntityToggle(getLocalStorageKey(entityId));
 
   useEffect(() => {
-    const newTargetEntities: TargetEntity[] = Object.values(selectedElements).map((item) => ({
-      id: item.id,
-      entity_type: item.entity_type ?? '',
-      name: item.name ?? item.observable_value ?? item.representative?.main.toString() ?? '',
-    }));
+    const newTargetEntities: TargetEntity[] = Object.values(selectedElements).map((item) => {
+      const name = item.name ?? item.observable_value ?? item.attribute_abstract ?? item.explanation ?? item.representative?.main ?? '';
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx]: Shouldn't it be the getMainRepresentative method used here ?

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx [hunk 2]
```diff
@@ -708,11 +708,14 @@ const StixCoreRelationshipCreationFromEntity: FunctionComponent<StixCoreRelation
...
+      return {
+        id: item.id,
+        entity_type: item.entity_type ?? '',
+        name: String(name),
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx]: ?

Comment (ValentinBouzinFiligran) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx]: did this to make TypeScript ok

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx (hunk 1 above)
Comment (ValentinBouzinFiligran) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx]: good idea

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx (hunk 2 above)
Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.tsx]: Ok, we might revert it with the correct use of getMainRepresentative :)

**Review by SarahBocognano — APPROVED** [review]
Tested ✅

**Pull request merged**